src/api/Nhs.Appointments.Persistance/Models/NotificationConfigurationDocument.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Newtonsoft.Json;
+
+namespace Nhs.Appointments.Persistance.Models;
+
+[CosmosDocumentType("notification_configuration")]
```

Comment:
As mentioned earlier I think the type for this could be system with an id of notification_configuration